### PR TITLE
use timer instead of fuse (fixes #87)

### LINKED ISSUE #87: switch to require(timer) instead of fuse
Gregg says Fuse is intended for doing things once.

### PR DESCRIPTION
**Pull request by mozkeeler**

### CODE REVIEW

File: lib/monitor.js [hunk 1]
```diff
@@ -29,24 +32,26 @@ exports.recordEvent = function recordEvent(eventName) {
...
+  if (!storage.lastUploadMillis ||
+      storage.lastUploadMillis + UPLOAD_INTERVAL < nowMillis) {
+    storage.lastUploadMillis = nowMillis;
+    let options = { simulate: false }
+    if (!myprefs.enable_reporting) {
+      options.simulate = true;
+    }
+    // We might lose events between when this.upload() returns and this.clear()
+    monitor.stop();
+    monitor.upload(UPLOAD_URL, options).
```
Comment (monicachew) [inline lib/monitor.js:48]: return the promise, in case we need to use it later

**Comment by monicachew** [comment]
Looks good!

File: lib/monitor.js (hunk 1 above)
Comment (mozkeeler) [inline lib/monitor.js:48]: Let me know if this isn't what you meant...

**Comment by monicachew** [comment]
Looks good, thanks.

**Pull request merged**